lib/client.go

```diff
@@ -395,6 +413,17 @@ func GetWorkspaceVar(organization, wsName, key string) (*Var, error) {
 
 // GetVarsFromWorkspace returns a list of Terraform variables for a given workspace
 func GetVarsFromWorkspace(organization, workspaceName string) ([]Var, error) {
+	orgExists, err := OrganizationExists(organization)
+	if err != nil {
+		return []Var{}, fmt.Errorf("GetVarsFromWorkspace: organization is required")
+	} else if !orgExists {
+		return []Var{}, fmt.Errorf("GetVarsFromWorkspace: invalid organization %s", organization)
+	}
+
+	if workspaceName == "" {
+		return []Var{}, fmt.Errorf("GetVarsFromWorkspace: workspace is required")
```

Comment:
```suggestion
		return nil, fmt.Errorf("GetVarsFromWorkspace: workspace is required")
```
(Same in other similar lines.)